restore.c

```diff
@@ -1230,9 +1230,9 @@ readTimeLineHistory(TimeLineID targetTLI)
 	elog(DEBUG, "the calculated branch history is as below;");
 	for (i = 0; i < parray_num(result); i++)
 	{
-		pgTimeLine *timeline = parray_get(result, i);
+		pgTimeLine *timeline2 = parray_get(result, i);
```

Comment:
Do we need the line? Is `s/timeline2->tli/(parray_get(result, i))->tli/` enough?